.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,44 @@
+<!--
+Please make sure you've read and understood our contributing guidelines;
+https://github.com/aws/amazon-ecs-agent/blob/master/CONTRIBUTING.md
+
+Please provide the following information:
+-->
+
+### Summary
+<!-- What does this pull request do? -->
+
+### Implementation details
+<!-- How are the changes implemented? -->
+
+### Testing
+<!-- How was this tested? -->
+<!--
+Note for external contributors:
+`make short-test` and `make test` can run anywhere in a development environment
+like your laptop.  Please ensure both of these pass before opening the pull
+request.  `make run-functional-tests` must be run on an EC2 instance with an
+instance profile allowing it access to AWS resources.  Running
+`make run-functional-tests` may incur charges to your AWS account; if you're
+unable or unwilling to run these tests in your own account, we can run the tests
+and provide test results.
+-->
+- [ ] Builds (`make release`)
+- [ ] Unit tests (`make short-test`) pass
+- [ ] Integration tests (`make test`) pass
+- [ ] Functional tests (`make run-functional-tests`) pass
+
+New tests cover the changes: <!-- yes|no -->
+
+### Description for the changelog
+<!--
+Write a short (one line) summary that describes the changes in this
+pull request for inclusion in the changelog:
```

Comment:
I like this [commit](https://github.com/aws/amazon-ecs-agent/commit/c9aefebc2b3007f09468f651f6308136bd7b384f).